etc/test-config.js

```diff
@@ -94,5 +97,14 @@ module.exports = {
         authType: 'BASIC',
         rejectUnauthorized: false,
         ssl:      true
+    },
+testConnection: {
+	    host:     testHost,
+        port:     restPort,
+        user:     testUser,
+        password: testPassword,
+        authType: restAuthType,
+        rejectUnauthorized: false,
+        ssl:      true
```

Comment:
Should it be possible to use this test without an SSL connection?

Reply:
Yes. The tests are running fine without ssl field.